### PR TITLE
Fix self host .env examples and documentation

### PR DESCRIPTION
**Pull request by aboutphilippe**
Fix docs, .env and docker compose to have api run and connect to local supabase

### CODE REVIEW

**Comment by topher-lo** [comment]
Thanks a ton for the PR! Will review and get back to you. Can you confirm this works on your machine?

Congrats on being our first public PR. We should probably send you some Tracecat merch

**Comment by aboutphilippe** [comment]
I can confirm the API talks to db and frontend talks to API properly.

image

**Comment by daryllimyt** [comment]
Hey, thanks so much for this PR! I will look at this in a bit as well, just debugging our staging env atm

File: docs/installation.mdx [hunk 1]
```diff
@@ -82,68 +83,75 @@ We recommend you grab a free static url from their page.
...
 
 ## Start Tracecat
+
 ### Backend
+
 At this point, the Supabase stack should already be running. Run the following to bring up Tracecat `nginx`, `api`, and `runner` services:
+
 ```bash
-docker compose -f docker-compose.self-host.yaml up
+docker compose -f up
```
Comment (daryllimyt) [inline docs/installation.mdx]:
```suggestion
docker compose up
```

**Review by daryllimyt — CHANGES_REQUESTED** [review]
LGTM! Just one change in the `docker compose up` step - don't need the -f flag. Thanks so much for this :)

**Pull request merged**